packages/driver/src/cypress/command_queue.ts

```diff
@@ -60,336 +60,322 @@ const commandRunningFailed = (Cypress, state, err) => {
   })
 }
 
-export default {
-  create: (state, timeout, whenStable, cleanup, fail, isCy) => {
-    const queue = $queue.create()
-
-    const { get, slice, at, reset, clear, stop } = queue
-
-    const logs = (filter) => {
-      let logs = _.flatten(_.invokeMap(queue.get(), 'get', 'logs'))
+export class CommandQueue extends Queue<Command> {
+  state: any
+  timeout: any
+  whenStable: any
+  cleanup: any
+  fail: any
+  isCy: any
```

Comment:
Should the types match what is [documented](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts#L398-L399)?

Reply:
You're right. I'll handle this in the part 3 of this PR series.